infrastructure/resource_group_level/modules/webapps/trade-finance-manager-api-no-calculated-variables.bicep

```diff
@@ -96,3 +96,4 @@ module tfmApiWebapp 'webapp.bicep' = {
 }
 
 output defaultHostName string = tfmApiWebapp.outputs.defaultHostName
+output tfmApiAppSettings object = appSettings
```

Comment:
This is a bit crude, I set the appSettings on tfm-api and then output them here, and then in `...calculated-variables.bicep` create a union with the outputted appSettings and calculated values & reset the appSettings on tfm-api

Reply:
I don't think you should set them here at all - just output them.

Reply:
Sounds good I'll update, I wasn't sure if anything needed to be set when we create the app which is why I did it this way